src/data/guidelines/color-tokens.js

```diff
@@ -902,6 +925,27 @@ const colorTokens = {
         },
       },
     },
+    '$hover-ui-light': {
+      role: ['$“Light” variant hover'],
```

Comment:
```suggestion
      role: ['"Light" variant hover'],
```

not sure which kind of quotation marks they're supposed to be but we probably don't need the `$` either right?

Reply:
oh woops. yeah. they should be the curly quotes